### ISSUE TITLE
Feature request: non-Nvidia GPUs support

### ISSUE THREAD

**Issue by timofonic** [wookayin/gpustat#29]
Hello.
Hello Please consider it, the name is quite global and would make many geeks happier. Linux ecosystem needs more global apps and good wrappers to make it more standarized. In the future maybe someone would be able to do a pure C implementation and even not require proprietary binaries? Whold it be possible? :D

**Comment by wookayin** [comment]
Yeah, I agree with your opinion. Actually we *no more* depend on a proprietary binary (nvidia-smi) because we've recently switched to use nvidia's C API, but it might be also possible to add some non-nvidia backend (e.g. AMD series) afterwards. But I haven't tried that so far.

**Comment by sebma** [comment]
@wookayin Hi, 
Let me thank you for the great work you've done so far.

Can you please add in your README.md the fact that your tools only works for Nvidia GPUs because it's not obvious for everyone ?

Thanks.
